SmartDeviceLinkTests/SDLFileManagerMultipleFilesSpec.m

```diff
@@ -0,0 +1,603 @@
+//
+//  SDLFileManagerMultipleFilesSpec.m
+//  SmartDeviceLink-iOS
+//
+//  Created by Nicole on 8/17/17.
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "SDLDeleteFileResponse.h"
+#import "SDLError.h"
+#import "SDLFile.h"
+#import "SDLFileManager.h"
+#import "SDLFileType.h"
+#import "SDLListFiles.h"
+#import "SDLListFilesOperation.h"
+#import "SDLListFilesResponse.h"
+#import "SDLNotificationConstants.h"
+#import "SDLPutFile.h"
+#import "SDLPutFileResponse.h"
+#import "SDLRPCResponse.h"
+#import "SDLUploadFileOperation.h"
+#import "TestMultipleFilesConnectionManager.h"
+#import "TestProgressResponse.h"
+#import "TestResponse.h"
+
+QuickSpecBegin(SDLFileManagerMultipleFilesSpec)
```

Comment:
I know it's ugly, but we only do one spec per class, so this should go into the `SDLFileManagerSpec` as a new describe at the bottom.

Reply:
Fixed.